v1.1/datareference/datacite.xml

```diff
@@ -2,82 +2,683 @@
+                    <property>
+                        <name>schemeURI</name>
+                        <type>url</type>
+                        <definition>URI of the affiliation identifier schema.</definition>
+                    </property>
+                </section>
+            </section>
+        </section>
+
+        <!-- #3 Title -->
+        <section>
+            <name>titles</name>
+            <type>DataCite</type>
+
+            <section>
+                <name>title #</name>
+                <type>DataCite</type>
+                <definition>A name or title by which a resource is known. May be the title of a dataset or the name of a piece of software.</definition>
+
+                <property>
+                    <name>title</name>
+                    <type>string</type>
+                    <definition>Value of title</definition>
+                </property>
+                <property>
+                    <name>titleType</name>
+                    <type>string</type>
+                    <value>[AlternativeTitle, Subtitle, TranslatedTitle, Other]</value>
+                    <definition>The type of Title.</definition>
+                </property>
+            </section>
+        </section>
+
+        <!-- #4 Publisher -->
         <property>
-            <name>DataDOI</name>
-            <definition>A unique Digital object identifier of data. In the current context a URI linking to the published dataset.</definition>
-            <type>url</type>
+            <name>publisher</name>
+            <type>string</type>
+            <definition>The name of the entity that holds, archives, publishes prints, distributes, releases, issues, or produces the resource. This property will be used to formulate the citation, so consider the prominence of the role. For software, use Publisher for the code repository. If there is an entity other than a code repository, that "holds, archives, publishes, prints, distributes, releases, issues, or produces" the code, use the property Contributor/contributorType/hostingInstitution for the code repository.</definition>
         </property>
 
+        <!-- #5 PublicationYear -->
         <property>
-            <name>ResourceType</name>
-            <definition>A description of the resource. The format is open, but the preferred format is a single term of some detail so that a pair can be formed with the sub-property.Text formats can be free-text OR terms from the CASRAI Publications resource type list. In the odML context the value will mostly be 'Dataset'.</definition>
+            <name>publicationYear</name>
             <type>string</type>
+            <definition>The year when the data was or will be made publicly available.  In the case of resources such as software or dynamic data where there may be multiple releases in one year, include the Date/dateType/dateInformation property and sub-properties to provide more information about the publication or release date details.</definition>
```

Comment:
"The year in which"
sub-properties is not entirely clear to me

Reply:
This is the official datacite definition, I don't want to tamper with it.